bawan-shna-abdulrahman/src/Components/Main.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+// import RenderMovie from "./RenderMovie";
+import MovieGrid from "./MovieGrid";
+
+export default function Main(props) {
+  // const query = props.query;
+  const movies = props.movies;
```

Comment:
you can use the destructured here like this
```suggestion
  const {movies} = props
```

Reply:
yeh thank you didnt know about that